doc/classes/ProjectSettings.xml

```diff
@@ -403,19 +403,19 @@
 			Sets the window to full screen when it starts.
 		</member>
 		<member name="display/window/size/height" type="int" setter="" getter="" default="600">
-			Sets the main window height. On desktop, this is the default window size. Stretch mode settings use this also as a reference when enabled.
+			Sets the game's main viewport height. On desktop platforms, this is the default window size. Stretch mode settings also use also as a reference when enabled.
```

Comment:
`also use also` should be `also use this`

Reply:
Wow, can't believe I didn't pick up on that. Fixed by #32555.

Reply:
Great